### ISSUE TITLE
showRSS.info is transitioning to new.ShowRSS.info

### ISSUE THREAD

**Issue by garfield69** [SchizoDuckie/DuckieTV#655]
The ShowRSS searchProvider is a strange beast.
Queries for `torrentName S00E00` are first translated to `torrentName 00x00` because that is the naming regime showRSS provides.
However, torrentDialog and autoDownload are not accepting *any* results from the showRSS searchProvider because the *strict* filterByScore expects the query string to match the torrent names.

Query `torrentName S00E00`  != torrent `torrentName 00x00`.

While researching this, I came across this: [http://showrss.info/?cs=blog&id=48](http://showrss.info/?cs=blog&id=48)
```
[...]
So, there we go: there's a new showRSS! You can try it today by browsing to new.showrss.info. Users with accounts on the old showRSS can migrate their account to the new showRSS by using the "Migrate Account" area in the new website. Just enter your current showRSS username and password, and a fresh account with the same username and password will be created in the new showRSS, and most of the show selections will be copied. Before doing so, please read the instructions carefully. Contact us on Twitter if you have questions or run into issues.
[...]
This current website you're reading this on right now will eventually be phased out, but there's no date planned at this moment. When this happens, the new showRSS will take the place of this service, and migration will still be possible for a while. You will not need to migrate or register again if you have already done so.
[...]
```

So in addition to fixing the *no-results-bug* for showRSS, I will switch to using new.showRSS.info and adjust the screen scraping logic of the showRSS serviceProvider.

TLDR: I'm going to fix the showRSS service provider because its broken.
